src/transport_ofi.c

```diff
@@ -95,6 +95,7 @@ static shmem_transport_ctx_t** shmem_transport_ofi_contexts = NULL;
 static size_t shmem_transport_ofi_contexts_len = 0;
 static size_t shmem_transport_ofi_grow_size = 128;
 
+#define SHMEM_CTX_DEFAULT_ID -1
```

Comment:
Don't we get this from `shmem.h`?

Reply:
Moved it from shmem.h to transport_ofi.c in e85e895.  Shouldn't need to expose this to users.

Reply:
Whoops!  I see it's moved.  Good plan.